_includes/parse-server/file-adapters.md

```diff
@@ -175,6 +175,94 @@ var api = new ParseServer({
 
 Don't forget to change **S3_ACCESS_KEY**, **S3_SECRET_KEY** and **S3_BUCKET** to their correct value.
 
+##### S3Adapter configuration for Digital Ocean Spaces
+
+Spaces is an S3 equivalent prodivided by Digital Ocean. It's use the same api as S3 so you can use it with the S3 Adapter.
```

Comment:
It's not an equivalent (that would be judgmental); let's stick with "S3-compatible" for all S3-compatible services here.

> [Digital Ocean Spaces](https://try.digitalocean.com/cloud-storage) is a S3-compatible storage service in the cloud. See their [documentation](https://docs.digitalocean.com/products/spaces/) for more details.

Reply:
This is not resolved but I'll comment on the new changes.